TeachingRecordSystem/src/TeachingRecordSystem.Api/V2/Handlers/GetOrCreateTrnRequestHandler.cs

```diff
@@ -164,7 +186,8 @@ public async Task<TrnRequestInfo> Handle(GetOrCreateTrnRequest request, Cancella
             Status = status,
             QtsDate = qtsDate,
             PotentialDuplicate = status == TrnRequestStatus.Pending,
-            SlugId = request.SlugId
+            SlugId = request.SlugId,
+            AccessYourTeachingQualificationsLink = trnToken is not null ? Option.Some($"{_accessYourQualificationsOptions.BaseAddress}/qualifications/start?trn_token={trnToken}") : default
```

Comment:
Now we have `/qualifications/start` repeated in a few places I wonder whether we should put that onto `AccessYourQualificationsOptions` - maybe `public const string StartUrlPath = "/qualifications/start`

Reply:
Done